int_experian/int_experian_sgjc/cartridge/templates/default/edq/verificationEngine.isml

```diff
@@ -3,12 +3,14 @@
 TASK:101728 Change Validate Button
 </iscomment>
 <isif condition="${URLUtils.continueURL().toString().search('Address') != -1}">
-	<button class="apply-button" type="button" id="form-submit"><span>${Resource.msg('global.apply','locale',null)}</span></button>
```

Comment:
Removed classes since it was taking the classes from the submit buttons and disappearing the form once it was pressed.